src/CommitmentsV2/SFA.DAS.CommitmentsV2/Data/Configuration/DataLockStatusConfiguration.cs

```diff
@@ -26,9 +26,9 @@ public void Configure(EntityTypeBuilder<DataLockStatus> builder)
                 .HasForeignKey(d => d.ApprenticeshipId)
                 .OnDelete(DeleteBehavior.ClientSetNull);
 
-            builder.HasOne(d => d.ApprenticeshipUpdate)
-                .WithMany(p => p.DataLockStatus)
-                .HasForeignKey(d => d.ApprenticeshipUpdateId);
+            //builder.HasOne(d => d.ApprenticeshipUpdate)
+            //    .WithMany(p => p.DataLockStatus)
+            //    .HasForeignKey(d => d.ApprenticeshipUpdateId);
```

Comment:
My understanding was this relationship to DataLocks is not needed, can I remove the block of code?

Reply:
Yes, I think so. This relationship relates to a really old mechanism that is no longer used and which we can probably get rid of in all forms now.